src/containers/PackingOverview/PackingPage/PackingPage.js

```diff
@@ -1,62 +1,363 @@
 import React, { Component } from "react";
-// import axios from "axios";
-// import BASEURL from "../../../services/backendUrlConnect";
-// import UnpackedItem from "../../../components/UnpackedItem/UnpackedItem";
+import axios from "axios";
+import BASEURL from "../../../services/backendUrlConnect";
+import AddItem from "../../../components/AddItem/AddItem";
 import BagSelector from "../../../components/BagSelectorCard/BagSelectorCard";
+import Bag from "../../../components/Bag/Bag";
 import "./PackingPage.css";
 
-// const bag = (items = []) => {
-//   return (
-//     <div>
-//       <div className="row px-3 col-6 col-md-4">
-//         {items.map((e, i) => {
-//           return (
-//             <div className="col-6 col-sm-6 col-md-4 p-0" key={i}>
-//               {/* <UnpackedItem /> */}
-//             </div>
-//           );
-//         })}
-//       </div>
-//     </div>
-//   );
-// };
-
 export default (class PackPage extends Component {
```

Comment:
Yo, this file is 451 lines long. That's too long. It's going to be a nightmare trying to work with it down the road. Trust me when I implore you to split it up! 😅 

Some suggestions for things that _could_ help with that _(haven't tried any of them locally)_:

* Move some or all of the logic from `componentDidMount` into a separate function in a separate file
* Come to think of it, in general, any function on a React component > 5-10 lines is probably a good candidate for splitting out - e.g. `executeDelete`, `unPack`, etc. here.
* Each `name` handled in `handleOnClick` is a good candidate for something that should be moved into a separate component.
* Any time you find it necessary to write a comment such as `{/*  BAG SELECTOR  */}`, it's probably a sign you should split what's under that into a new component - even if that component just takes in some props and renders them without logic.

I can help you with this if you're interested when we meet tomorrow. In the meantime, one thing I find help helpful with splitting up components that seem to touch a lot is to:

1. Completely remove any code about _modifying_ or _figuring out how to modify_ state, then forget it ever existed
2. Split up components by responsibility: so this component's `render` would just return a root container, maybe a couple of `div`s/whatnot within that, then entirely other components 
3. Move back any logic about _figuring out how to modify_ state into lower level components where possible
4. Move back any logic about _modifying_ state into lower level components where possible
5. Shove the rest of the things back into this root-level component

Reply:
Thank you for the feedback, loving it!  I cleaned up what I could. We're currently in crunch mode to try and get as much done before MVP deadline tomorrow morning. I will do my best to be neater moving forward.

Reply:
Seems reasonable. Anything is worth it when the alternative is missing a required deadline!

Still down to help if you want after the deadline!